sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -525,6 +524,358 @@ These APIs are invoked via DigitalTwinsClient.
 ```
 </details>
 
+## Digital Twins
+<details><summary><b>Terminology</b></summary>
+
+A digital twin is an instance of one of your custom-defined models.
+
+</details>
+
+<details><summary><b>Async APIs</b></summary>
+
+These APIs are invoked via DigitalTwinsAsyncClient.
+
+```java
+/**
+ * Gets a digital twin.
+ *
+ * @param digitalTwinId The Id of the digital twin. The Id is unique within the service and case sensitive.
+ * @return The application/json digital twin
+ */
+@ServiceMethod(returns = ReturnType.SINGLE)
+public Mono<string> getDigitalTwin(String digitalTwinId)
```

Comment:
I think we need another one with 
`public T Mono<T> getDigitalTwin(String digitalTwinId, Class<T> classType)`

Reply:
Ok now I understood so we need both

Mono<T> and Mono<Response<T>> as well.